### ISSUE TITLE
`sudo minidcos vagrant web` is not working in Release 2019.04.18.2

### ISSUE THREAD

**Issue by rajcspsg** [dcos/dcos-e2e#1732]
Hi,

I have Ubuntu 16.04 LTS system. I have installed minidcos version Release 2019.04.18.2.

```
$minidcos vagrant doctor
$minidcos vagrant download-installer
$minidcos vagrant create ./dcos_generate_config.sh --agents 0

```

All the above command are working fine. When I run `$minidcos vagrant  web` I get below error -

```
$sudo minidcos vagrant web
Traceback (most recent call last):
  File "minidcos", line 4, in <module>
  File "site-packages/click/core.py", line 764, in __call__
  File "site-packages/click/core.py", line 717, in main
  File "site-packages/click/core.py", line 1137, in invoke
  File "site-packages/click/core.py", line 1137, in invoke
  File "site-packages/click/core.py", line 956, in invoke
  File "site-packages/click/core.py", line 555, in invoke
  File "site-packages/dcos_e2e_cli/dcos_vagrant/commands/web.py", line 32, in web
  File "site-packages/dcos_e2e_cli/dcos_vagrant/commands/_common.py", line 215, in cluster
  File "site-packages/dcos_e2e/cluster.py", line 133, in from_nodes
  File "site-packages/dcos_e2e/cluster.py", line 103, in __init__
  File "</tmp/_MEImjTBfz/decorator.pyc:decorator-gen-2>", line 2, in _wait_for_ssh
  File "site-packages/retry/api.py", line 74, in retry_decorator
  File "site-packages/retry/api.py", line 33, in __retry_internal
  File "site-packages/dcos_e2e/cluster.py", line 49, in _wait_for_ssh
  File "site-packages/dcos_e2e/node.py", line 498, in run
  File "site-packages/dcos_e2e/_node_transports/_ssh_transport.py", line 139, in run
  File "site-packages/dcos_e2e/_common.py", line 150, in run_subprocess
subprocess.CalledProcessError: Command '['ssh', '-o', 'IdentitiesOnly=yes', '-o', 'StrictHostKeyChecking=no', '-i', '/tmp/dd105f3c394948af89011f3f88990bd9/f79d5869106f45bba918a67150baad09/dcos-e2e-06a6cf6c-d08b-4e43-a43f-a3c318e23907/.vagrant/machines/dcos-e2e-06a6cf6c-d08b-4e43-a43f-a3c318e23907-master-0/virtualbox/private_key', '-l', 'vagrant', '-o', 'PreferredAuthentications=publickey', '-o', 'UserKnownHostsFile=/dev/null', '-o', 'LogLevel=QUIET', '172.28.128.11', '/bin/sh', '-c', "'systemctl status sshd.socket || systemctl status sshd'"]' returned non-zero exit status 255.
[9469] Failed to execute script minidcos

```

The partial strace log  is [here](https://gist.github.com/rajcspsg/024ab1e8525e5357dd44d47b1b33372b).

When I uninstalled 2019.04.18.2 version and installed older 2019.04.08.1, then the web console is working fine.

**Comment by adamtheturtle** [comment]
This is interesting. It looks like perhaps the cluster was not up when you first tried and then while downgrading the cluster happened to come up. Can you please try again but use `minidcos vagrant wait` before `minidcos vagrant web`?

**Comment by rajcspsg** [comment]
@adamtheturtle - I'm tested minidcos version 2019.04.25.0. ` sudo minidcos vagrant wait` command gives the same error.

```
$ sudo minidcos vagrant wait
Traceback (most recent call last):
  File "/usr/local/bin/minidcos", line 10, in <module>
    sys.exit(minidcos())
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/dcos_e2e_cli/dcos_vagrant/commands/wait.py", line 43, in wait
    cluster=cluster_vms.cluster,
  File "/usr/local/lib/python3.7/site-packages/dcos_e2e_cli/dcos_vagrant/commands/_common.py", line 215, in cluster
    public_agents=set(map(self.to_node, self.public_agents)),
  File "/usr/local/lib/python3.7/site-packages/dcos_e2e/cluster.py", line 136, in from_nodes
    cluster_backend=backend,
  File "/usr/local/lib/python3.7/site-packages/dcos_e2e/cluster.py", line 106, in __init__
    _wait_for_ssh(node=node)
  File "</usr/local/lib/python3.7/site-packages/decorator.py:decorator-gen-2>", line 2, in _wait_for_ssh
  File "/usr/local/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/usr/local/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/usr/local/lib/python3.7/site-packages/dcos_e2e/cluster.py", line 52, in _wait_for_ssh
    shell=True,
  File "/usr/local/lib/python3.7/site-packages/dcos_e2e/node.py", line 508, in run
    capture_output=capture_output,
  File "/usr/local/lib/python3.7/site-packages/dcos_e2e/_node_transports/_ssh_transport.py", line 139, in run
    pipe_output=capture_output,
  File "/usr/local/lib/python3.7/site-packages/dcos_e2e/_common.py", line 152, in run_subprocess
    stderr=stderr,
subprocess.CalledProcessError: Command '['ssh', '-o', 'IdentitiesOnly=yes', '-o', 'StrictHostKeyChecking=no', '-i', '/private/tmp/c40f75a8a10941fbb305fabb11712c91/268319b1b509478db30e6451aaf16e2d/dcos-e2e-31743584-2b82-4acf-9347-e82cc069f54a/.vagrant/machines/dcos-e2e-31743584-2b82-4acf-9347-e82cc069f54a-master-0/virtualbox/private_key', '-l', 'vagrant', '-o', 'PreferredAuthentications=publickey', '-o', 'UserKnownHostsFile=/dev/null', '-o', 'LogLevel=QUIET', '172.28.128.35', '/bin/sh', '-c', "'systemctl status sshd.socket || systemctl status sshd'"]' returned non-zero exit status 255.
```

**Comment by Vince-Nieuw** [comment]
I have this issue as well in the latest release (2019.05.01.0) running in a python 3.7 virtual environment. 

```
(dcos-installer) ➜  dcos-installers minidcos vagrant wait --cluster-id default
Traceback (most recent call last):
  File "/Users/vince/dcos-venv/dcos-installer/bin/minidcos", line 10, in <module>
    sys.exit(minidcos())
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/dcos_e2e_cli/dcos_vagrant/commands/wait.py", line 43, in wait
    cluster=cluster_vms.cluster,
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/dcos_e2e_cli/dcos_vagrant/commands/_common.py", line 215, in cluster
    public_agents=set(map(self.to_node, self.public_agents)),
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/dcos_e2e/cluster.py", line 136, in from_nodes
    cluster_backend=backend,
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/dcos_e2e/cluster.py", line 106, in __init__
    _wait_for_ssh(node=node)
  File "</Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/decorator.py:decorator-gen-2>", line 2, in _wait_for_ssh
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/dcos_e2e/cluster.py", line 52, in _wait_for_ssh
    shell=True,
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/dcos_e2e/node.py", line 508, in run
    capture_output=capture_output,
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/dcos_e2e/_node_transports/_ssh_transport.py", line 139, in run
    pipe_output=capture_output,
  File "/Users/vince/dcos-venv/dcos-installer/lib/python3.7/site-packages/dcos_e2e/_subprocess_tools.py", line 145, in run_subprocess
    stderr=stderr,
subprocess.CalledProcessError: Command '['ssh', '-o', 'IdentitiesOnly=yes', '-o', 'StrictHostKeyChecking=no', '-i', '/private/var/folders/w6/j5t1g9694d79c52sc_rmhc500000gq/T/fd08fe6666b34d3c8bede47b1dde09d7/fe2c9b92b0b74cd5b551f2d10d58a108/dcos-e2e-ba971c5e-617a-437c-a363-4c7db7a7f94a/.vagrant/machines/dcos-e2e-ba971c5e-617a-437c-a363-4c7db7a7f94a-master-0/virtualbox/private_key', '-l', 'vagrant', '-o', 'PreferredAuthentications=publickey', '-o', 'UserKnownHostsFile=/dev/null', '-o', 'LogLevel=QUIET', '172.28.128.21', '/bin/sh', '-c', "'systemctl status sshd.socket || systemctl status sshd'"]' returned non-zero exit status 255.
```

**Comment by adamtheturtle** [comment]
Thanks all - this was a bug fixed in #1772 . The latest release should fix this - please re-open if that is not the case. Sorry for any hassle!
